conf/pacific/rados/6-node-cluster-6-clients.yaml

```diff
@@ -0,0 +1,94 @@
+# Rados tier-2 test configuration, small test config for basic tests
+# Deployment for all the ceph daemons , with 3 mon's, 3 mgr's, and 15 OSD daemons
+globals:
+  - ceph-cluster:
+      name: ceph
+      node1:
+        networks:
+          - provider_net_cci_12
+        role:
+          - _admin
+          - mon
+          - mgr
+          - installer
+          - node-exporter
+          - alertmanager
+          - grafana
+          - prometheus
+          - crash
+      node2:
+        networks:
+          - provider_net_cci_12
+        role:
+          - mon
+          - mgr
+          - mds
+          - rgw
+          - node-exporter
+          - alertmanager
+          - crash
+      node3:
+        networks:
+          - provider_net_cci_12
+        role:
+          - osd
+          - node-exporter
+          - crash
+        no-of-volumes: 5
+        disk-size: 25
+      node4:
+        networks:
+          - provider_net_cci_12
+        role:
+          - osd
+          - node-exporter
+          - crash
+        no-of-volumes: 5
+        disk-size: 25
+      node5:
+        role:
+          - osd
+          - node-exporter
+          - crash
+        no-of-volumes: 5
+        disk-size: 25
+      node6:
+        networks:
+          - provider_net_cci_12
+        role:
+          - mon
+          - mgr
+          - mds
+          - rgw
+          - node-exporter
+          - crash
+      node7:
+        networks:
+          - provider_net_cci_8
+        role:
+          - client
+      node8:
+        networks:
+          - provider_net_cci_8
+        role:
+          - client
+      node9:
+        networks:
+          - provider_net_cci_9
+        role:
+          - client
+      node10:
+        networks:
+          - provider_net_cci_9
+        role:
+          - client
+      node11:
+        networks:
+          - provider_net_cci_7
+        role:
+          - client
+      node12:
+        networks:
+          - provider_net_cci_7
+        role:
+          - client
```

Comment:
As per the test case, the clients should be 9 in the configuration the clients are 6.

Reply:
Configuring 9 clients would increase the number of nodes. So I have updated  the client count to 6, with which we can sufficiently test the feature. The same count is updated in doc too.

Let me know if you feel we would need 9 clients. Would increase the count.